dao-user-reference.adoc

```diff
@@ -0,0 +1,192 @@
+
+BSQ can only be created in one of two ways:
+
+1. through the genesis distribution (detailed below)
+2. through an issuance transaction (compensation request or reimbursement request)
+
+While BSQ is just bitcoin, it should only be stored in a Bisq wallet.
+
+[sidebar]
+.Warning
+--
+WARNING: **Please do not try to send BSQ to a non-Bisq wallet, or to otherwise create your own BSQ transactions**. The rules to form a valid BSQ transaction are rather complex, and even the slightest error can result in invalid BSQ. Even the <<dao/specification#,technical spec>> does not cover every single detail—the only complete reference is the source code. Also note that future code changes may break custom transactions or implementations. Bisq will NOT provide support for any such cases.
+--
+
+Ultimately, BSQ is stored in a Bitcoin wallet, but the Bisq software takes several measures to keep BSQ and BTC separate to prevent mixing the two by accident. On disk, BTC and BSQ wallets are stored in separate files. In the Bisq interface, BSQ wallets are prefixed with a 'B' to prevent users from mistakenly using BSQ as BTC.
+
+As an additional protection measure, Bisq wallets use a special BIP44 derivation path that other wallets won't understand.
+
+For user convenience, the built-in Bisq wallets for BTC and BSQ share the same seed.
+
+More details are in the <<dao/specification#,technical specification>>.
+
+### BSQ Genesis Distribution
+
+As you may have noticed, the Bisq DAO did not launch with a supply of zero BSQ tokens. So where did the initial supply of BSQ tokens come from?
+
+In short, the initial supply of BSQ tokens was created to account for all the contributions to the Bisq project before the mainnet launch of the Bisq DAO.
+
+These contributions can be divided into 2 segments.
+
+1. _Before October 2017_
++
+In the early days of the project, before Bisq tracked compensation requests and voting results, Bisq collected donations. Over time, about 25 BTC was collected from 144 people. In October 2017, as part of the implementation of the <<dao/phase-zero#the-bsq-token,Phase Zero>> plan, it was decided that 2.5 BTC of the total would be distributed back to these past contributors as BSQ where each person would receive an amount proportional to their prior contributions.
++
+2. _After October 2017_
++
+In October 2017, the Phase Zero plan went into effect. Subsequent contributions to Bisq were https://github.com/bisq-network/compensation/issues[documented on GitHub^] and votes were cast https://docs.google.com/spreadsheets/d/1xlXDswj3251BPCOcII-UyWlX7o7jMkfYBE-IZ5te5Ck[on this spreadsheet^] by stakeholders.
+
+The genesis BSQ distribution was the sum of both segments of past contributions.
+
+Ultimately, the genesis transaction issued https://explorer.bisq.network/[6,157,480 BSQ to 202 contributors^].
```

Comment:
"6,157,480 BSQ" is too high. It will be likely about 3.5-4M will know exact number soon

Reply:
I mean total 2.5M + about 1 - 1.5 M

Reply:
I made it 4 for now, but we can set the correct number when we know it.